src/slider/docs/demo.js

```diff
@@ -0,0 +1,5 @@
+var SliderDemoCtrl = function($scope) {
```

Comment:
Add a line for `angular.module('ui.bootstrap.slider').controller('SliderDemoCtrl', SliderDemoCtrl);`